### ISSUE TITLE
A one-off crash when checking playlist

### ISSUE THREAD

**Issue by MikkoMMM** [axcore/tartube#53]
I attempted to reproduce this twice, but it worked correctly the other times. Anyway, when I tried checking a playlist, there was this crash.

I figured I'd include the recently-used.xbel warnings as well since those didn't appear on the subsequent runs. I've disabled that functionality on my system by making it a directory but that's only ever produced warnings, not crashes.

```
(tartube:38061): Gtk-WARNING **: 18:54:16.152: Attempting to read the recently used resources file at '/home/mikko/.local/share/recently-used.xbel', but the parser failed: Error reading file “/home/mikko/.local/share/recently-used.xbel”: Is a directory.

(tartube:38061): Gtk-WARNING **: 18:54:21.798: Attempting to store changes into '/home/mikko/.local/share/recently-used.xbel', but failed: Failed to rename file “/home/mikko/.local/share/recently-used.xbel.8NOJE0” to “/home/mikko/.local/share/recently-used.xbel”: g_rename() failed: Is a directory

(tartube:38061): Gtk-CRITICAL **: 18:55:01.110: ../gtk/gtk/gtktreeview.c:6711 (validate_visible_area): assertion `has_next' failed.
There is a disparity between the internal view of the GtkTreeView,
and the GtkTreeModel.  This generally means that the model has changed
without letting the view know.  Any display from now on is likely to
be incorrect.

Segmentation fault (core dumped)
code 139
```

**Comment by axcore** [comment]
It is very difficult to reproduce these crashes, because they are (probably) Gtk bugs that occur almost randomly.

I have made a few changes in v1.4.0 that might prevent some of the crashes. If you see another crash, you can disabled a few minor cosmetic features, which are usually related to the crash. Click **Edit > System preferences... > General > Modules** and select **Assume that Gtk is broken...*

**Comment by axcore** [comment]
I believe this is fixed, as of v1.5.0, as long as 1. The most recent version of Gtk is installed on your system, or 2. You have enabled 'broken mode' by clicking **Edit > System preferences... > General > Modules > Assume that Gtk is broken, and disable some features**

(The problem does not apply on MS Windows. On Linux/BSD, broken mode is enabled by default for new installations.)
